cl_dll/tri.cpp

```diff
@@ -1,4 +1,4 @@
-//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
+//========= Copyright � 1996-2002, Valve LLC, All rights reserved. ============
```

Comment:
Broken character became broken slightly differently

Reply:
It was broken on the review page but it's visible in the comments...

Reply:
Even when you are completely sure that auto-formatting is disabled and encoding should not be changed, IDE goes brrrrrrrrrrrrrrrrrr

Thanks for noting actually, would to fix then

Reply:
Fixed